src/core/file_format/pkg.h

```diff
@@ -116,29 +119,56 @@ class PKG {
         return pkgSize;
     }
 
+    std::string GetPkgFlags() {
+        return pkgFlags;
+    }
+
     std::string_view GetTitleID() {
         return std::string_view(pkgTitleID, 9);
     }
 
+    PKGHeader GetPkgHeader() {
+        return pkgheader;
+    }
+
+    bool isFlagSet(u32_be variable, PKGContentFlag flag) {
+        return (variable) & static_cast<u32>(flag);
+    }
+
+    std::vector<std::pair<PKGContentFlag, std::string>> flagNames = {
```

Comment:
This seems to be constant, an extra less heap allocation :P
```suggestion
    static constexpr std::array<std::pair<PKGContentFlag, std::string_view>, 10> FlagNames = {
```